src/Uno.UI.TestComparer/AzureDevopsDownloader.cs

```diff
@@ -65,7 +65,12 @@ public async Task DownloadArtifacts(string basePath, string project, string defi
 				{
 					if (Path.GetDirectoryName(file) != fullPath)
 					{
-						File.Move(file, Path.Combine(fullPath, Path.GetFileName(file)));
+						var destFileName = Path.Combine(fullPath, Path.GetFileName(file));
+
+						if (!File.Exists(destFileName))
```

Comment:
There is no reason where we would need to update the target file ?

Reply:
The file will be the same, that can happen if a build is present twice in the list.